core/src/main/java/com/sequenceiq/cloudbreak/service/cluster/Package.java

```diff
@@ -0,0 +1,65 @@
+package com.sequenceiq.cloudbreak.service.cluster;
+
+import java.util.List;
+
+public class Package {
```

Comment:
I don't know if we ever log this class, but maybe worth adding a `toString` for this and `PackageName` too
thanks for extracting them!